platform/solr/solr-dependencies/pom.xml

```diff
@@ -23,6 +23,13 @@
     <name>DDF :: Platform :: Solr :: Dependencies</name>
     <description>Shaded jar of Solr core dependencies.</description>
     <packaging>jar</packaging>
+    <properties>
```

Comment:
Ran Owasp in the following modules (platform-standalone-server, solr-dependencies, solr-factory, and solr-xpath) and got the following results:                                                                       
Vulnerable Dependencies: 0
Vulnerabilities Found: 0
Vulnerabilities Suppressed: 0